From f9e6fc3befa2b077d312f664c9d94777cb1a2fa4 Mon Sep 17 00:00:00 2001
From: root <root@xenialbase>
Date: Sat, 16 Jul 2016 07:28:18 +0000
Subject: [PATCH 2/4] Less pagination for less annoyance

---
 config/initializers/kaminari_config.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/initializers/kaminari_config.rb b/config/initializers/kaminari_config.rb
index 3cbe9a0..46bb4fb 100644
--- a/config/initializers/kaminari_config.rb
+++ b/config/initializers/kaminari_config.rb
@@ -1,6 +1,6 @@
 Kaminari.configure do |config|
-  config.default_per_page = 20
-  config.max_per_page = 100
+  config.default_per_page = 300
+  config.max_per_page = 300
   # config.window = 4
   # config.outer_window = 0
   # config.left = 0
-- 
2.7.4

