From ba0b0fc7c95f163fab5cf98d8162c7d65702fe51 Mon Sep 17 00:00:00 2001
From: root <root@gitlab>
Date: Fri, 6 May 2016 09:18:58 +0000
Subject: [PATCH 04/12] Less pagination for less annoyance

---
 config/initializers/kaminari_config.rb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/initializers/kaminari_config.rb b/config/initializers/kaminari_config.rb
index 3cbe9a0..46bb4fb 100644
--- a/config/initializers/kaminari_config.rb
+++ b/config/initializers/kaminari_config.rb
@@ -1,6 +1,6 @@
 Kaminari.configure do |config|
-  config.default_per_page = 20
-  config.max_per_page = 100
+  config.default_per_page = 300
+  config.max_per_page = 300
   # config.window = 4
   # config.outer_window = 0
   # config.left = 0
-- 
1.9.1

